### PR TITLE
Fix Bug1387972 Set lang and dir attributes on l20n inputs

### PR DESCRIPTION
**Pull request by nathanielnebel**
Add dir, and lang attributes to l20n inputs https://bugzilla.mozilla.org/show_bug.cgi?id=1387972 .

### CODE REVIEW

File: pontoon/base/static/js/fluent_interface.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ var Pontoon = (function (my) {
           } else {
             cls = ' class="custom-attribute clearfix"';
             label = '<div class="wrapper">' +
-              '<input type="text" class="id" placeholder="enter-attribute-id" value="' + id + '">' +
+              '<input type="text" class="id" placeholder="enter-attribute-id" value="' + id + '" dir="' + Pontoon.locale.direction + '" data-script="' + Pontoon.locale.script + '"  lang="' + Pontoon.locale.code + '">' +
```
Comment (mathjazz) [inline pontoon/base/static/js/fluent_interface.js]: This element is actually used for the attribute `id`, so no need to add the attributes here.

File: pontoon/base/static/js/fluent_interface.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ var Pontoon = (function (my) {
                   '<label class="id built-in" for="ftl-id-' + this + '">' +
                     '<span>' + this + ' (e.g. </span><span class="stress">' + Pontoon.locale.examples[pluralForm] + '</span>)<sub class="fa fa-remove remove" title="Remove"></sub>' +
                   '</label>' +
-                  '<input class="value" id="ftl-id-' + this + '" type="text" value="' + value + '">' +
+                  '<input class="value" id="ftl-id-' + this + '" type="text" value="' + value + '" dir="' + Pontoon.locale.direction + '" data-script="' + Pontoon.locale.script + '"  lang="' + Pontoon.locale.code + '">' +
```
Comment (mathjazz) [inline pontoon/base/static/js/fluent_interface.js]: Double space in front of `  lang=`.

**Comment by nathanielnebel** [comment]
@mathjazz I agree with your suggestion for creating a function to generate the input element as to facilitate code reuse. The only reason I didn't do that was that the input element on line 72 didn't specify a maxlength (nor do I think a variable lives within that scope). The input element on line 99 does specify that value. If you would still believe that to be advisable I can create a function to do it and leave the parameter for max length optional and use that to return the appropriate element for each section.

**Comment by mathjazz** [comment]
Yeah, I'm aware that that input elements are not exactly the same. Let's create such function.

File: pontoon/base/static/js/fluent_interface.js (hunk 2 above)
Comment (nathanielnebel) [reply pontoon/base/static/js/fluent_interface.js]: Resolved, generation of this element is now contained in a function

File: pontoon/base/static/js/fluent_interface.js (hunk 1 above)
Comment (nathanielnebel) [reply pontoon/base/static/js/fluent_interface.js]: Oops, reverted it.

File: pontoon/base/static/js/fluent_interface.js [hunk 3]
```diff
@@ -164,6 +164,21 @@ var Pontoon = (function (my) {
 
 
       /*
+       * Generate Input value element
+       */
+      inputValueElement: function (id, value, maxLength) {
+         var base = '<input class="value" id="ftl-id-' + id + '" type="text" value="' + value + '"';
```
Comment (mathjazz) [inline pontoon/base/static/js/fluent_interface.js]: Nit: Use 2 spaces for indentation instead of 3.

**Comment by nathanielnebel** [comment]
@mathjazz Indentation fixed, does Mozilla have a styleguide for Javascript that I can read?

**Pull request merged**